include/seastar/core/chunked_fifo.hh

```diff
@@ -23,6 +23,7 @@
 
 #ifndef SEASTAR_MODULE
 #include <algorithm>
+#include <assert.h>
```

Comment:
how about using `cassert` in place of `assert.h`? i understand they are practically equivalent. and because `assert()` is always implemented as a macro, there is no such as thing like `std::assert()`. but, IMHO, would be nice if we can always use the headers provided by the C++ standard library when writing a C++ application/library. what do you think?

Reply:
Good point, will switch it over to `cassert`.